src/aliceVision/image/curves.hpp

```diff
@@ -0,0 +1,193 @@
+#pragma once
+
+#include <vector>
+
+namespace alicevision
+{
+    namespace image {
+
+        class avLUTf
+        {
+        protected:
+            // list of variables ordered to improve cache speed
+            int maxs;
+            float maxsf;
+            float* data;
```

Comment:
maybe use a std::vector<float>, it will get things easier managing the memory
```suggestion
            std::vector<float> data;
```